features/logging/elasticsearch.feature

```diff
@@ -86,3 +86,63 @@ Feature: Elasticsearch related tests
       | infra-00  |
       | .kibana_  |
       | .security |
+
+  # @author qitang@redhat.com
+  # @case_id OCP-28140
+  @admin
+  @destructive
+  Scenario: Elasticsearch retention policy testing
+    Given I switch to the first user
+    And I create a project with non-leading digit name
+    And evaluation of `project` is stored in the :proj clipboard
+    Given I obtain test data file "logging/loggen/container_json_log_template.json"
+    When I run the :new_app client command with:
+      | file | container_json_log_template.json |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | run=centos-logtest,test=centos-logtest |
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-logging" project
+    Given I obtain test data file "logging/clusterlogging/index_management_test.yaml"
+    When I create clusterlogging instance with:
+      | remove_logging_pods | true                       |
+      | crd_yaml            | index_management_test.yaml |
+    Then the step should succeed
+    Given I wait for the "infra" index to appear in the ES pod with labels "es-node-master=true"
+    And I wait for the "app" index to appear in the ES pod with labels "es-node-master=true"
+    When I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | _cat/indices?format=JSON |
+      | op           | GET                      |
+    Then the step should succeed
+    And evaluation of `@result[:parsed].select {|e| e['index'].start_with? "app"}.map {|x| x["index"]}` is stored in the :app_indices clipboard
+    And evaluation of `@result[:parsed].select {|e| e['index'].start_with? "infra"}.map {|x| x["index"]}` is stored in the :infra_indices clipboard
+    And evaluation of `@result[:parsed].select {|e| e['index'].start_with? "audit"}.map {|x| x["index"]}` is stored in the :audit_indices clipboard
+    
+    Given I successfully merge patch resource "elasticsearch/elasticsearch" with:
+      | {"spec": {"managementState": "Unmanaged"}} |
```

Comment:
Do we need to restore the change after the test ?

Reply:
The CR will be removed after the scenario([line 108](https://github.com/openshift/verification-tests/pull/1664/files#diff-f444ec1a915696d4912ef14fc13d41becb90880d8cad5d4540fa9aaa85a85adcR108)), so no need to restore the changes.

Reply:
Ack